Build.ps1

```diff
@@ -11,7 +11,7 @@ $suffix = @{ $true = ""; $false = "$branch-$revision"}[$branch -eq "master" -and
 foreach ($src in ls src/Serilog.*) {
     Push-Location $src
 
-    & dotnet pack -c Release -o ..\..\.\artifacts --version-suffix=$suffix
+    & dotnet pack -c Release -o ..\..\.\artifacts --version-suffix=$suffix --include-source
```

Comment:
Interesting - does `--include-source` make the resulting package more easily debuggable?